sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/messaging/eventhubs/implementation/core/listener/adapter/BatchMessagingMessageListenerAdapter.java

```diff
@@ -48,7 +48,7 @@ public void onMessage(EventBatchContext eventBatchContext) {
 
     @Override
     @SuppressWarnings("unchecked")
-    public AbstractAzureMessageConverter<EventBatchContext, EventData> getMessageConverter() {
-        return (AbstractAzureMessageConverter<EventBatchContext, EventData>) super.getMessageConverter();
+    public AbstractJacksonAzureMessageConverter<EventBatchContext, EventData> getMessageConverter() {
```

Comment:
Does this have to be an `AbstractJacksonAzureMessageConverter`?

Reply:
Updated to the interface `AzureMessageConverter`.